.github/workflows/ci.yml

```diff
@@ -0,0 +1,39 @@
+name: CI
+
+# About email notifications, see <https://git.io/JUo27>.
+
+on:
+  push:
+    # Run in PRs and for bors, but not on master.
+    branches:
+      - 'auto'
+      - 'try'
+      - '!gh-pages'
+  pull_request:
+    branches:
+      - 'master'
+  # TODO: fill the cron here
+  # schedule:
+  #   - cron: "30 0 * * 1" # Every Monday at half past midnight UTC skip-pr skip-stable
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    env:
+      RUST_BACKTRACE: 1
+      PROJECT_NAME: miri
```

Comment:
What is this environment variable for?

Reply:
from appveyor template. I should have check that it is unused.